### PR TITLE
doc: add initial table for coverage

### PR DESCRIPTION
**Pull request by mhdawson**
Add table that shows which of the benchmarks
run nightly cover the user cases or key attributes

### CODE REVIEW

**Comment by mhdawson** [comment]
@nodejs/benchmarking  please review/comment

File: docs/case_coverage.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * [key attributes](https://github.com/nodejs/benchmarking/blob/master/docs/runtime_attributes.md)
+
+# Use Cases
+
+Use case                                   |Benchmark with coverage| Notes
+-------------------------------------------|-----------------------|-------
+Service oriented architectures (SOA) | |
+Microservice-based applications | |
+Generating/serving dynamic web page content | Acme Air Ops/s <BR>Acme Air Latency |
+SPA applications with bidirectional communication over WebSockets and/or HTTP/2 | |
```
Comment (ofrobots) [inline docs/case_coverage.md]: It would be expand out SPA. I haven't seen it used too frequently (but maybe my viewfinder is too narrow?)

Comment (gareth-ellis) [inline docs/case_coverage.md]: It would also mean we don't end up saying "Single page application applications"

**Comment by mhdawson** [comment]
Address first set of comments.

**Comment by gareth-ellis** [comment]
LGTM

**Comment by mhdawson** [comment]
landed as ad4cb5e7fddcd6100e666b22dc59c1f883c06560

**Pull request closed without merging**